velox/functions/sparksql/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ add_library(
   ArraySort.cpp
   Bitwise.cpp
   CompareFunctionsNullSafe.cpp
```

Comment:
Would you move `CompareFunctionsNullSafe.cpp` to `Comparisons.cpp`?

Reply:
Yes I think we should put them in same file

Reply:
@Yuhta @jinchengchenghh , `equalnullsafe` is now moved into cmp functions family. Please take a further look.